src/views/index.ejs

```diff
@@ -5,25 +5,32 @@
     <link rel="stylesheet" href="/stylesheets/style.css" />
   </head>
   <body>
+
+  <div id="navigation-style" >
     <%- include partials/nav %>
+    </div>
     <h1><%= title %></h1>
     <% if(success) { %>
-    <p style="color:forestgreen"><%= message %></p>
+    <p id="green-message"><%= message %></p>
     <% } else { %>
-    <p style="color: red"><%= message %></p>
+    <p id="red-message"><%= message %></p>
     <% } %>
-    <p>Upload any file</p>
+    <p>Upload any file:</p>
+
     <form
       id="uploadFileForm"
       action="/upload"
       method="post"
       enctype="multipart/form-data"
     >
+          <label class="upload-button" for="fileItem">CHOOSE FILE</label>
+      <%= file.name %>
```

Comment:
@marcodeanda This will not load the UI since no variable is supplied from the backend while rendering the EJS. Can this be fixed via the following snippet: (Add an element with the ID `#file-name`)

```js
document.querySelector("#uploadFileForm").onchange = function(){
  document.querySelector("#file-name").textContent = this.files[0].name;
}
```